populate.js

```diff
@@ -14,7 +14,6 @@ const populateDb = () => {
 
 
 const savePhotos = async (primaryUrls, productPhotosUrls) => {
-
   let dbRecords = [];
   let featuresPhotoSizes = [[960, 832], [960, 400], [960, 123], [960, 832], [700, 568], [700, 568], [700, 568], [960, 832], [547, 454], [300, 270], [300, 270], [300, 270], [50, 50], [50, 50], [50, 50], [50, 50], [50, 50]];
```

Comment:
Is the purpose of defining photo size to generate images from http://placeimg.com/ ?
Are these the dimensions required for features section ?

Reply:
That's correct, I went over this with Justin. I'll provide his pictures in this order size for him to make a consistent grid for all products.